index.html

```diff
@@ -85,14 +85,15 @@ <h4>Developers "re-inventing the wheel" is potentially harmful</h4>
 
   <section>
     <h2>Description</h2>
-    <section data-dfn-for="WindowOrWorkerGlobalScope">
-      <h3>Extensions to the <code>WindowOrWorkerGlobalScope</code> interface</h3>
+    <section data-dfn-for="Crypto">
+      <h3>Extensions to the <code>Crypto</code> interface</h3>
       <p>
-        This document extends the <code><dfn data-cite="!HTML#windoworworkerglobalscope">WindowOrWorkerGlobalScope</dfn></code>
-        interface defined by [[!HTML]].
+        The <dfn data-cite="WebCryptoAPI/#dfn-Crypto">Crypto</dfn> interface is
+        defined in [[!WebCryptoAPI]].
       </p>
       <pre class="idl">
-        partial interface mixin WindowOrWorkerGlobalScope {
+        [Exposed=(Window,Worker)]
+        partial interface mixin <dfn id="dfn-Crypto"></dfn>Crypto</dfn> {
```

Comment:
Crypto isn't a mixin, so you just want `partial interface`:

```suggestion
        partial interface <dfn id="dfn-Crypto"></dfn>Crypto</dfn> {
```